### PR TITLE
[Docs] Style, grammar, spelling, and capitalization updates

### PR DESCRIPTION
**Pull request by nikhil-vytla**
## Change Description:
- Update styling, grammar and fix spelling, capitalization across public-facing docs
- Add references/hyperlinks for popular attribution approaches
- Remove typos
## Why?
- Readability, consistency

### CODE REVIEW

File: docs/trulens_explain/attribution_parameterization.md [hunk 1]
```diff
@@ -60,30 +62,32 @@ Now suppose you want to know the attribution of some internal layer on the final
...
-Eg: Integrated gradients is a sample from a straight line from a baseline to a value.  
 **Cuts needed:** Define a DoI that samples from the default attribution cut.  
 **Model inputs** → InputCut  
 **DoI/Attribution Cut** → Your baseline/DoI/attribution layer, anchor:'in'  
 **QoI Cut** → OutputCut
  
 #### Case 5: Internal explanations
 
-**Use case:** You want to explain an internal layer. Things like integrated gradients are a DoI on the baseline to the value, but it is on the layer the baseline is defined.
+**Use case:** You want to explain an internal layer. Methods like Integrated Gradients are a DoI on the baseline to the value, but it is located on the layer the baseline is defined.
```
Comment (nikhil-vytla) [inline docs/trulens_explain/attribution_parameterization.md:90]: Note: the wording here was initially confusing, looking for feedback on how to improve readability here.

File: docs/trulens_explain/attribution_parameterization.md [hunk 2]
```diff
@@ -96,47 +100,47 @@ If you want to explain an internal layer, you do not move the DoI layer.
...
 **What is a Distribution of Interest (DoI)?**
 
 The distribution of interest is a concept of aggregating attributions over a sample or distribution. 
 
-* GradCam does this over a guassian distribution of inputs. 
-* Shapley values do this over different background data. 
-* Integrated gradients does this over an interpolation from a baseline to the input.
+* [Grad-CAM](https://ieeexplore.ieee.org/document/8237336) does this over a Gaussian distribution of inputs. 
+* [Shapley values (SHAP)](https://shap.readthedocs.io/en/latest/example_notebooks/overviews/An%20introduction%20to%20explainable%20AI%20with%20Shapley%20values.html) do this over different background data. 
+* [Integrated Gradients](https://arxiv.org/abs/1703.01365) do this over an interpolation from a baseline to the input.
```
Comment (nikhil-vytla) [inline docs/trulens_explain/attribution_parameterization.md:134]: Looking for feedback on specific references to point to here. Is it more digestable to point to implementations/tutorials/packages, to academic papers, or to both?

Comment (joshreini1) [reply docs/trulens_explain/attribution_parameterization.md:134]: Both is my vote. Maybe in parenthesis like: Grad-CAM (Paper, Implementation/Tutorial/Package)?

File: docs/how-to-use.html [hunk 3]
```diff
@@ -9,7 +9,7 @@
     <meta name="googlebot" content="index,follow,snippet,archive">
 
     <title>TruLens: Explainability for LLM Apps</title>
-    <meta name="author" content="Maros Janota">
```
Comment (nikhil-vytla) [inline docs/how-to-use.html:12]: Note: Fairly certain this is a remnant from a template. Updating to "TruEra" as this person is neither affiliated with TruEra or TruLens.

**Review by rshih32 — APPROVED** [review]

**Pull request merged**